lib/core/src/model.rs

```diff
@@ -1871,6 +1877,23 @@ macro_rules! get_invoice_description {
     };
 }
 
+#[macro_export]
+macro_rules! get_invoice_destination_pubkey {
```

Comment:
What's the motivation to use a macro here? Seems like a function would work

Reply:
You're right, fixed [18a7025](https://github.com/breez/breez-sdk-liquid/pull/649/commits/18a7025f792fb1a7480b7845e973a05c006dad83)